### ISSUE TITLE
Help training the network

### ISSUE THREAD

**Issue by Kyanite123** [1adrianb/face-alignment#116]
Hi Adrian, 

I run you network on some testing data and the results are amazing!! 
I would like to train your network on another dataset with a different number of points.
Is it possible for you to supply the training code? This would great!

Thanks, 
Alex

**Comment by pharrellyhy** [comment]
Hi,

Have you figured out how to train the net? When I trained the net, the losses are around 4e-4 at the first few iterations and the validation loss is almost the same which I think the network doesn't learn anything useful. Thanks.
